default.vcl

```diff
@@ -61,6 +61,8 @@ sub vcl_recv {
         # Routing preset here as vulcan is unable to route on query strings
     } elseif (req.url ~ "\/content\?.*isAnnotatedBy=.*") {
         set req.http.Host = "public-content-by-concept-api";
+    } elseif (req.url ~ "\/concept\/search.*$") {
```

Comment:
Is this too much of a wildcard? This will capture everything that begins `/concept/search` so `/concept/search-mongo?asdad=asas` or  `/concept/searchCoolNewService`. I think you should probably go for something like `"\/concept\/search[\?].*$"`.
Also, if you have anything in your search like `/concept/search?term=a&isAnnotatedBy=XXX` it will get captured by the rule above it.